lib/swagger-client.js

```diff
@@ -1555,7 +1555,7 @@ var Property = function(name, obj, required) {
   if(obj.$ref)
     this.$ref = simpleRef(obj.$ref);
   else if (obj.type === 'array' && obj.items) {
-    if(obj.items.$ref)
+    if(obj.items)
       this.$ref = simpleRef(obj.items.$ref);
     else
       obj = obj.items;
```

Comment:
This seems like a bad change to me. This conditional will _never_ hit the else statement because it's already wrapped in a conditional that checks for the existence of `obj.items`. The original code seems appropriate, because then the entire block reads:

> Does `obj.items` contain a $ref? If it does, then retain the resolved $ref to `this`. Otherwise, the schema attached to `obj.items` must contain the information `this` needs.